vstruct/defs/minidump.py

```diff
@@ -570,7 +570,7 @@ def __init__(self, bytez):
                 stream = vars(self)[sclass.__name__] = sclass()
                 stream.vsParse(bytez, offset=soffset)
             else:
-                logging.info('unknown stream type')
+                logging.info(f'Unknown stream type of {header.StreamType}')
```

Comment:
i would discourage you from using f-strings for logging.

when you use an f-string, the interpreter will always create a new string instance with the formatted results, while using the `logging` routines will conditionally format and create the string if the logging level is high enough. while it may be not particularly relevant for `INFO` messages that always get shown, this is an important performance consideration for logging within tight loops, `DEBUG` messages, etc. therefore i'd recommend sticking to a single style that avoids f-strings.

Reply:
Aye. You're right. Just personally force of habit having me to reach for f-strings since they're super handy dandy :laughing: